app/models/recurring_todos/abstract_repeat_pattern.rb

```diff
@@ -174,7 +174,7 @@ def continues_recurring?(previous)
     # same day as the previous
     def determine_start(previous, offset=0.day)
       start = self.start_from || NullTime.new
-      now = Time.zone.now
+      now = Time.now
```

Comment:
I'm looking at this commit: https://github.com/TracksApp/tracks/commit/3647d795875c0126d91e8151ee49e583bf2a174e maybe is relevant?